### ISSUE TITLE
Crash with v1.5.3

### ISSUE THREAD

**Issue by Phil57** [martabal/qbittorrent-exporter#93]
Hello, after updating to 1.5.3, the exporter crashes

```qbittorrent-exporter (version v1.5.3
Author: martabal
Using log level: INFO
[2024-12-14 08:50:25] INFO qbittorrent URL: xxxxxxx
[2024-12-14 08:50:25] INFO username: xxxxxxxxxxx
[2024-12-14 08:50:25] INFO password: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
[2024-12-14 08:50:25] INFO Features enabled: [Trackers, Label with hash (experimental)]
[2024-12-14 08:50:25] INFO Started
[2024-12-14 08:50:25] INFO Starting the exporter
panic: a previously registered descriptor with the same fully-qualified name as Desc{fqName: "qbittorrent_torrent_total_downloaded_bytes", help: "The current total download amount of torrents (in bytes)", constLabels: {}, variableLabels: {name,hash}} has different label names or a different help string
goroutine 36 [running]:
github.com/prometheus/client_golang/prometheus.(*Registry).MustRegister(...)
	/go/pkg/mod/github.com/prometheus/client_golang@v1.20.5/prometheus/registry.go:406
qbit-exp/prometheus.registerGauge(0xc000113b20, 0xc0001a22d0)
	/app/prometheus/prometheus.go:432 +0x31a
qbit-exp/prometheus.Torrent(0xc0002020f0, 0xc0001a22d0)
	/app/prometheus/prometheus.go:152 +0x147b
qbit-exp/qbit.getData(0xc0001a22d0, 0xc44370, 0x1, 0xc000078310)
	/app/qbit/qbit.go:114 +0x27d
created by qbit-exp/qbit.AllRequests in goroutine 7
	/app/qbit/qbit.go:219 +0x1df
qbittorrent-exporter (version v1.5.3)```

**Comment by martabal** [comment]
Thanks for the report, fixed with [efd9d44](https://github.com/martabal/qbittorrent-exporter/commit/efd9d44ab3bacdecbc808fd532010a6de236e9f5) and released in [v1.5.4](https://github.com/martabal/qbittorrent-exporter/releases/tag/v1.5.4)

**Comment by Phil57** [comment]
I confirm it's working. Thanks
